From fa72f3d95b593b72c8b97c375e2c4996c7cb6d9f Mon Sep 17 00:00:00 2001
From: "Wang, ArvinX" <arvinx.wang@intel.com>
Date: Thu, 24 Jan 2019 15:33:41 +0800
Subject: [PATCH] Fix the "Add new user?" text is not visible

Text is not visible due to use the dark text color.

Change-Id: If34729a2d2e4c2aac6e811d0a4659de05d592f8c
Tracked-On: https://jira.devtools.intel.com/browse/OAM-75448
Signed-off-by: Wang, ArvinX <arvinx.wang@intel.com>
---
 .../base/core/res/res/layout/alert_dialog_title_material.xml    | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/car_product/overlay/frameworks/base/core/res/res/layout/alert_dialog_title_material.xml b/car_product/overlay/frameworks/base/core/res/res/layout/alert_dialog_title_material.xml
index 0071d5c..16524cd 100644
--- a/car_product/overlay/frameworks/base/core/res/res/layout/alert_dialog_title_material.xml
+++ b/car_product/overlay/frameworks/base/core/res/res/layout/alert_dialog_title_material.xml
@@ -51,7 +51,7 @@
             android:layout_centerVertical="true"
             android:layout_marginStart="@dimen/car_keyline_1"
             android:layout_marginEnd="@dimen/car_keyline_1"
-            style="@style/CarTitle"/>
+            style="@style/CarTitle.Light"/>
     </RelativeLayout>
 
     <Space
-- 
2.20.0

